### ISSUE TITLE
Problem when reloading library

### ISSUE THREAD

**Issue by tomas-mhsl** [nokia/RED#203]
I have a robot referenced library, written in python and I added two more methods. These methods are not automatically discovered by RED, then I tried to reload the library but I get the following error:
"Problems occurred during library specification generation:
Illegal base64 character 20"
If I run some tests, these methods are used without problems but they are not discovered by RED.
I  tried to remove the library and reload it, but the problem is still there

**Comment by michalanglart** [comment]
Hi,

what are the versions of RED/Robot and in particular Python you're using?

**Comment by tomas-mhsl** [comment]
Python 2.7.13
Robot Framework 3.0.2 (Python 2.7.13 on win32)
RED - Robot Editor version 0.8.4
Eclipse Platform Version: Photon (4.8)

**Comment by michalanglart** [comment]
Do you use non-ASCII characters in your source? I've seen similar behavior when:

- using some polish unicode letters in library source code (in comments)
- using python 2.7.x
- NOT declaring source encoding as described in [PEP 263](https://www.python.org/dev/peps/pep-0263/)

Please check if adding following line at the top of your source would help:

```
# -*- coding: UTF-8 -*-
```

**Comment by tomas-mhsl** [comment]
I didn't find any strange character.
I have tried setting the coding to UTF-8 (also to latin-1) and same result, error is still there

**Comment by adam-wypych** [comment]
You can try to use some editors like Notepad++ and encode/convert your files to be UTF-8.

**Comment by tomas-mhsl** [comment]
Using Notepad++ I have tried to convert to UTF-8 (it was previously as UTF-8), ANSI, change EOL to avoid mixed EOL in the file, etc... but always occurs the same result, when I reload the library, the same error appears

**Comment by tomas-mhsl** [comment]
I finally was able to find the line where the issue occurs. In the __init__ of my library I have the following code:
`        try:
            self._SL = BuiltIn().get_library_instance('SeleniumLibrary')
        except (RobotNotRunningError, RuntimeError) as err:
            log.warn(err)
            self._SL = None`
I have to add the try-except to allow the library to be correctly loaded (if the try-catch doesn't exist it returns an error). When I add the try-except it works if I just assign the None value to self._SL, but when I add the log.warn(err) it reports the "Illegal base64 character 20". "log" is from the import:
`from robot.api import logger as log`
It seems that the problem is into robot.api.logger ... I have just removed the log.warn from the __init__ code and now the library can be reloaded...
